### PR TITLE
Alignment on dragging

### PR DESCRIPTION
**Pull request by akaDuality**

### CODE REVIEW

**Comment by akaDuality** [comment]
@drucelweisse I'm working on tests, but code can be reviewed

File: Document/Sources/Document/Drawing/DrawingController.swift [hunk 1]
```diff
@@ -200,3 +207,10 @@ extension CGPoint {
         return false
     }
 }
+
+extension CGPoint {
+    static func -(lhs: Self, rhs: Self) -> Self {
+        Self(x: lhs.x - rhs.x,
+             y: lhs.y - rhs.y)
+    }
+}
```
Comment (drucelweisse) [inline Document/Sources/Document/Drawing/DrawingController.swift:216]: maybe move to separate file for CGPoint

**Review by drucelweisse — APPROVED** [review]
Lgtm! Maybe in meantime move some basic extensions to CGPoint like - operator, in comment up there

File: Document/Sources/Document/Drawing/DrawingController.swift (hunk 1 above)
Comment (akaDuality) [reply Document/Sources/Document/Drawing/DrawingController.swift:216]: Yeah. Already moved, but it will be in #41

**Comment by akaDuality** [comment]
So aligned possible now!

Screen Shot 2022-07-29 at 01 11 17

**Pull request merged**